gammapy/estimators/points/tests/test_lightcurve.py

```diff
@@ -738,6 +738,43 @@ def test_recompute_ul():
         lightcurve.recompute_ul(n_sigma_ul=4)
 
 
+@requires_data()
+def test_dataset_stat_type():
+    datasets = get_spectrum_datasets()
+    selection = ["all"]
+    estimator = LightCurveEstimator(
+        energy_edges=[1, 3, 30] * u.TeV, selection_optional=selection, n_sigma_ul=2
+    )
+    lightcurve = estimator.run(datasets)
+    assert_allclose(
+        lightcurve.dnde_ul.data[0], [[[3.260703e-13]], [[1.159354e-14]]], rtol=1e-3
+    )
+
+    lightcurve_dataset = FluxPointsDataset(
+        data=lightcurve, models=[lightcurve.reference_model]
+    )
+    lightcurve_dataset.stat_type = "profile"
```

Comment:
Why tests are needed here when they are tests in test_flux_points?

Reply:
see comment above